crates/flox/src/commands/general.rs

```diff
@@ -272,52 +203,82 @@ pub struct ConfigSetBool {
     value: String,
 }
 
-/// bug in bpaf (<https://github.com/pacak/bpaf/issues/171>)
+// / bug in bpaf (<https://github.com/pacak/bpaf/issues/171>)
```

Comment:
```suggestion
/// bug in bpaf (<https://github.com/pacak/bpaf/issues/171>)
```

Reply:
those should go entirely, also messes up the docs for some reason